changelog/deprecate_delete.dd

```diff
@@ -0,0 +1,10 @@
+The `delete` keyword has been deprecated.
+
+See the $(LINK2 $(ROOT_DIR)deprecate.html#delete, Deprecated Features) for more information.
+
+Starting with this release, using the `delete` keyword will result in a deprecation warning.  The
+warning is scheduled to remain for 1 year, after which it will be changed to an error.  The
+error is then to remain for 1 year, after which `delete` will be removed as a keyword.
```

Comment:
Are you aware of Jack's new deprecation DIP?
https://github.com/dlang/DIPs/pull/108
One of the things that are on the list is instead of having a fixed date, a fixed version is better for users because it's important which version will stop supporting it (there might be a few delay of publishing a version, e.g. LDC or distros in general)

Reply:
Yes, I've seen the DIP, but it needs to go through a review and approval process which I think will take time.  Given the the fact that we don't have a well-defined deprecation process, I made something up, and that's what you see here.  Feel free to suggest an alternative wording, or maybe I should just omit the schedule altogether so we don't have to commit to anything.  I don't have an opinion either way.

Reply:
Yes, please omit the schedule.
In my experience, we most likely won't stick to it anyways.

Reply:
Done.